core-bundle/src/Resources/contao/pages/PageRoot.php

```diff
@@ -51,7 +54,7 @@ public function generate($rootPageId, $blnReturn=false, $blnPreferAlias=false)
 	 */
 	public function getResponse($rootPageId)
 	{
-		return new RedirectResponse($this->getRedirectUrl($rootPageId));
+		return new RedirectResponse($this->getRedirectUrl($rootPageId), Response::HTTP_SEE_OTHER);
```

Comment:
Are you sure this is the correct code? I remember we had a long discussion about this and changed it at some point. Cannot find the relevant issue anymore though.

Reply:
That's a very good question, and I think it depends 😂 I've actually changed this in 352b10b because the functional tests expect a 303 response. However, some places (e.g. `PageRoot::getResponse()` would have returned a 302, but I think that method was never used …

Reply:
Then I would suggest to stick to the default Symfony code (302) instead of explicitly defining it?

Reply:
Yes. 👍🏻

Reply:
Changed in 38324ccc195a25a0d4fe3654512d66cf2d37b2c6

Reply:
But this line still explicitly defines it. Shouldn't it be left as it was, as it is also unrelated to the PR?

Reply:
It sure should.

Reply:
Reverted in 4963145ed257d7f9b588096b9868cf7cf672091e.